counter/src/main.rs

```diff
@@ -0,0 +1,63 @@
+#[macro_use]
+extern crate serde;
+
+use oasis_std::{Context, Address};
+use std::str::FromStr;
+
+#[derive(Debug, Eq, PartialEq, Serialize, Deserialize, failure::Fail)]
+pub enum Error {
+    #[fail(display = "Only deployed multisig service can make this call.")]
+    NotAllowed,
+}
+
+#[derive(oasis_std::Service)]
+struct MultisigCounter {
+    count: u32,
+    allowed: Address,
+}
+
+impl MultisigCounter {
+    pub fn new(_ctx: &Context, allowed: String) -> Self {
+        Self {
+            count: 0,
+            allowed: Address::from_str(&allowed).unwrap(),
```

Comment:
this can panic. why not just accept an `Address` and not even deal with runtime overhead?